docs/modules/ROOT/pages/wan-replication.adoc

```diff
@@ -1,54 +1,124 @@
-= Synchronizing Clusters
-:description: To synchronize data between two or more clusters, you can use WAN Replication.
+= Synchronizing Data Between Clusters
+:description: You can use WAN replication to synchronize data between two or more clusters, and to transfer data from one cluster to another.
 
 {description}
 
-WAN Replication is a tool that can do *Active-Active* and *Active-Passive* replication. You can also configure WAN Replication parameters according to your use-case.
+== Use Cases
+
+WAN replication supports the following use cases:
+
+* Use WAN replication with a xref:failover-clients-with-hazelcast-cloud.adoc[failover client] as part of your disaster recovery strategy. WAN replication automatically synchronizes data between clusters. If a cluster fails, the failover client can switch to an alternative cluster with minimal disruption to service.
+
+* Migrate data to a new cluster before decommissioning your original cluster.
+
 
-For a complete reference of this feature, see the xref:hazelcast:wan:wan.adoc[Hazelcast Platform documentation].
 
 == Terminology
 
-[cols="e,a"]
+[cols="a,a"]
+|===
+|Term|Description
+
+| _Publisher_ cluster, also known as the _active_ cluster
+|Sends or transfers data to another cluster using WAN replication
+
+| _Consumer_ cluster, also known as the _passive_ cluster
+|Receives data from another cluster using WAN replication
 |===
-|Term|Definition
 
-|Publisher Cluster
-|The cluster that sends data within WAN Replication
+You can set up pairs of clusters to use either one or two-way replication.
+
+[cols="a,a"]
+|===
```

Comment:
I think we can merge this into the first table rather than having two tables with the same headings.

Reply:
Updated as requested.